client/app/app.html

```diff
@@ -2,7 +2,7 @@
     <div class="alert alert-warning mb-0 rounded-0"
```

Comment:
+alert-dismissible -> Alert has a close button (and in general for all others alerts with a close button)